src/pages/dashboardUserEdit/components/AddUserForm.tsx

```diff
@@ -216,7 +216,10 @@ export default function AddUserForm({
           : undefined,
         confirmEmail: !values.confirmEmail
           ? requiredError
-          : values.confirmEmail !== values.email
+          : values.confirmEmail !== values.email &&
+            values.email &&
+            values.confirmEmail.toLocaleLowerCase().toLocaleUpperCase() !==
+              values.email.toLocaleLowerCase().toLocaleUpperCase()
```

Comment:
or lowerCase or upperCase :D
```suggestion
            values.confirmEmail.toLocaleLowerCase() !==
              values.email.toLocaleLowerCase()
```